conf/deployment/ibmcloud/ibm_cloud_managed_3az_0m_3w_vpc_cluster.yaml

```diff
@@ -0,0 +1,31 @@
+
+# This is the basic config for IBM cloud usage
+---
+ENV_DATA:
+  platform: 'ibm_cloud'
+  deployment_type: 'managed'
+  region: 'eu-de'
+  zone: 'eu-de-3'
+  provider: "vpc-gen2"
+  worker_instance_type: "bx2.16x64"
+  master_replicas: 0
+  # Number of worker nodes per zone
```

Comment:
will this config creates 3*3  nodes in cluster? from creation logic it looks like it creates 3w nodes per cluster.

Reply:
In this configuration it will create 3 worker zones in total like it's in other platform configs, e.g.
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/deployment/aws/ipi_3az_rhcos_3m_3w.yaml

The comment on line 12 is leftover from previous config which I forgot to remove, because before - and logic of IBM Cloud deployment was to really take this replica as replica of nodes per AZ - which means without code change it will do 9 nodes in 3 zones, but I changed the logic to work like in other platforms, just forgot to remove this comment which is now misleading as you are pointing out. Let me send a PR to remove this comment.